packages/clerk-js/src/ui/components/SignIn/SignIn.tsx

```diff
@@ -128,6 +129,11 @@ function SignInRoutes(): JSX.Element {
               >
                 <LazySignUpVerifyPhone />
               </Route>
+              {signInContext.withSessionTasks && (
+                <Route path='add-organization'>
```

Comment:
❓  Do we want to pull this in from `SESSION_TASK_ROUTE_BY_KEY` ?

Reply:
Makes sense! I'll have to refactor a bit of that constant lookups since it's value have a `/` concatenated on the string, and the `Route` path shouldn't include it